src/main/java/knightminer/simplytea/core/config/Teapot.java

```diff
@@ -18,6 +22,9 @@ public Teapot(ForgeConfigSpec.Builder builder) {
     milk_cow = builder.comment("If true, cows can be milked using a teapot to fill it with milk")
                       .translation("simplytea.config.teapot.milk_cow")
                       .define("milk_cow", true);
+    teapot_capacity = builder.comment("Amount of fluid a teapot can contain")
```

Comment:
Since we are not doing emptying, lets make this say something like "Amount of fluid consumed when filling a teapot from a tank". Contain suggests you can empty it into a tank

Reply:
Done.